variables.tf

```diff
@@ -9,6 +9,12 @@ variable "lacework_agent_name" {
   default     = "lacework-agent"
 }
 
+variable "lacework_agent_interface_connection_size" {
+  type        = string
+  description = "Desired value for the InterfaceConnectionSize Lacework agent parameter"
+  default     = "10000"
```

Comment:
Since I was told to use 10000, I placed 10000 as our default value. That being said, what should be the default value?

Reply:
LOL!!!! I just asked this same thing! (Amazing minds JP :hi-5:)